docs/Projects/TAD/ADRs/0003-tad-tool.md

```diff
@@ -0,0 +1,52 @@
+# ADR-0003 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+We are considering tooling for organizations to get more grip on their algorithms.
+Tooling for, for instance bias and fairness tests, and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+
+Transparency, we think, can be fostered by sharing reports from such a tool in a [standardized way](../Reporting-Standard/latest.md).
+
+There are several existing open source
+[tools](../Existing-Tools/Comparison/tools/) which
+we [have assessed](../Existing-Tools/Comparison/tools_comparison/).
+Some support only assessments, others already combine more features and can generate a report.
+There is however no tool
+that supports all the [requirements](../Existing-Tools/Comparison/requirements/) we have.
+
+These are our main requirements:
```

Comment:
I would propose to add a requirement like:

> The tool offers an option, including approval system, to publish algorithms using a subset of the information from a system card in public registers such as The Dutch Algorithm Register.

Reply:
While adding this I remember having discussions on the 'single source of truth'.. so if the register should point to the system card rather than having the system card (information) uploaded to the register.. so.. I am not sure what applies (and if maybe this should be added later?) @anneschuth  @robbertbos

Reply:
@uittenbroekrobbert I agree that with the idea that the single source of truth should be the project where the AI System Card is hosted, so this requirement is a little bit confusing. Let's ignore it for now.